### ISSUE TITLE
Design for navigating menus with keyboard

### ISSUE THREAD

**Issue by lynwilhelm** [simularium/simularium-planning#116]
## Use Case
The menus cannot be accessed today by keys only. The menu interaction needs to be updated to support better accessibility.

## Acceptance Criteria
User navigating website with keyboard should be able to use return key or spacebar to prompt a menu open from buttons

**Comment by lynwilhelm** [comment]
@blairlyons I don't believe any design work is needed beyond the button styling outlined in the design link provided. This work is based on how the code is written, designating appropriate css styles for button:hover, button:focus, etc. Or, it's possible these buttons use JavaScript. Happy to talk this out with a developer as needed though if expected behavior is unclear! https://xd.adobe.com/view/75b6a3c8-f650-443b-bccc-d1fcd134a35e-20e5
